api-reference/10 UI Components/dxSplitter/7 Interfaces/dxSplitterItem/size.md

```diff
@@ -5,7 +5,75 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the size of an item (pane) in pixels or as a percentage. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(() => {
+        $("#splitter").dxSplitter({
+            items: [
+                {
+                    // ...
+                    size: "50%",
+                }
+            ],
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-splitter ... >
+        <dxi-item ...
+            size="50%"
+        >
+        </dxi-item>
+    </dx-splitter>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSplitter ... >
+            <DxItem ... 
+                size="50%"
+            />
+        </DxSplitter>
+    </template>
+
+    <script setup>
+    import { DxSplitter, DxItem } from "devextreme-vue/splitter";
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from "react";
+    import Splitter, { Item } from "devextreme-react/splitter";
+
+    const App = () => (
+        <React.Fragment>
+            <Splitter ... >
+                <Item ... 
+                    size="50%"
+                />
+            </Splitter>
+        </React.Fragment>
+    );
+
+    export default App;
+
+---
+
+If you do not specify pane sizes, the UI component splits up the panes automatically with even distribution.
+
+[note]
+
+- The **size** value must not exceed [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#maxSize) and be less then [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#minSize).
```

Comment:
I don't like the phrase "must not exceed". What will happen if I do try to assign a value that's out of bounds? Size will remain unchanged? Or maybe the size will be set to closest limit? 

If we don't want to answer that question, I guess it's better to still use imperative mood. "Use max and min to specify contstraints."

```suggestion
- You can use [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#minSize) and [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Interfaces/dxSplitterItem/#maxSize) properties to specify size constraints.
```